python/bridge/chips/gap.py

```diff
@@ -32,8 +32,8 @@
 
 class gap_debug_bridge(debug_bridge):
 
-    def __init__(self, config, binaries=[], verbose=False):
-        super(gap_debug_bridge, self).__init__(config=config, binaries=binaries, verbose=verbose)
+    def __init__(self, config, binaries=[], verbose=False, fimages=[]):
```

Comment:
Also this prototype should not change, this argument should be passed with the flash image